docs(express/connect): Add performance options to express and connect (#9755)
getsentry/sentry-docs: docs/platforms/javascript/guides/express/index.mdx

const app = express();

// Sentry must be initialized before doing anything else with it
Sentry.init({
  dsn: "___PUBLIC_DSN___",

  // Adding Performance Monitoring by setting tracesSampleRate and adding integration
  // Set tracesSampleRate to 1.0 to capture 100% of transactions
  // We recommend adjusting this value in production
  tracesSampleRate: 1.0,

>   integrations: [new expressIntegration({ app })],
>   // You can also specify the routes you want to trace by adding `router: someRouter` instead of `app`

Reviewer: ```suggestion

  integrations: [expressIntegration({ app })],
  // It is possible to specify the routes that should be traced by passing a router instance to the `router` option:
  // expressIntegration({ router: someRouterInstance })
```

Reviewer: Note the removed `new` !

});

> // The request handler must be the first middleware on the app

Reviewer: ```suggestion
// The Sentry request handler middleware must be added before any other handlers
```

app.use(Sentry.Handlers.requestHandler());

> // Performance Monitoring: creating a trace for every incoming request

Reviewer: ```suggestion
// Performance Monitoring: Create spans and traces for every incoming request
```

## Monitor Performance

> By setting `tracesSampleRate` and adding the tracing middleware with `expressIntegration` tracing with Sentry is enabled in Express.

Reviewer: ```suggestion
You can monitor the performance of your Express application by setting a `tracesSampleRate`, adding the `expressIntegration`, and registering the `tracingHandler` middleware.
```
